src/WatcherDelegatingHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace k8s
+{
+    internal class WatcherDelegatingHandler : DelegatingHandler

```

Comment:
Perhaps document what this class does here?

Reply:
fixed